com.unity.render-pipelines.universal/Runtime/ForwardRenderer.cs

```diff
@@ -307,6 +326,7 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
             // But if we only require it for post processing or the scene camera then we do it after rendering transparent objects
             m_CopyDepthPass.renderPassEvent = (!requiresDepthTexture && (applyPostProcessing || isSceneViewCamera)) ? RenderPassEvent.AfterRenderingTransparents : RenderPassEvent.AfterRenderingOpaques;
             bool createColorTexture = RequiresIntermediateColorTexture(ref cameraData);
```

Comment:
Move the line `createColorTexture = (rendererFeatures.Count != 0);` to above code and then this one:

```suggestion
            createColorTexture |= RequiresIntermediateColorTexture(ref cameraData);
```